### ISSUE TITLE
Indent false negative with comments on first line of a block

### ISSUE THREAD

**Issue by golopot** [eslint/eslint#12145]
**Tell us about your environment**

* **ESLint Version:** 6.2.1
* **Node Version:** 12
* **npm Version:** 6

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
/* eslint indent: [2, 2] */

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
/* eslint indent: [2, 2] */

export function f() {
// a
  return 1
};

{
// a
  console.log(3)
}
```

https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IGluZGVudDogWzIsIDJdICovXG5cbmV4cG9ydCBmdW5jdGlvbiBmKCkge1xuLy8gYVxuICByZXR1cm4gMVxufTtcblxue1xuLy8gYVxuICBjb25zb2xlLmxvZygzKVxufVxuIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo5LCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7ImpzeCI6dHJ1ZX19LCJydWxlcyI6e30sImVudiI6e319fQ==

**What did you expect to happen?**
Report 2 errors. 

**What actually happened? Please include the actual, raw output from ESLint.**
0 errors.

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by kaicataldo** [comment]
Sorry this fell through the cracks. This is currently intentional behavior, because it’s possible that a comment could be referring to the line above and intentionally be aligned with it.

**Comment by kaicataldo** [comment]
Closing this since it's working as intended.
